authors/apps/articles/tests/test_data.py

```diff
@@ -58,3 +58,23 @@
 auth_error = "Authentication credentials were not provided."
 
 not_found = "Not found"
+
+valid_rating = {  
+               "ratings":5
+}
+
+invalid_rating = {  
+               "ratings":7
+}
+
+invalid_rating2 = {  
+               "ratings":-1
```

Comment:
This oversight is a mark against the quality of code you're writing.
Ensure you fix it.

Reply:
I have fixed this too